modoptions.lua

```diff
@@ -1050,19 +1050,19 @@ local options={
 	},
 
 	{
-		key    = 'teamcolors_anonymous_mode',
-		name   = 'Anonymous Mode',
-		desc   = 'Anonimizes players in the match so you have harder time telling who is who.',
-		type   = 'list',
+		key     = 'teamcolors_anonymous_mode',
+		name    = 'Anonymous Mode',
+		desc    = "Anonymize players by changing colors (based on chosen mode) and replacing names with question marks, making it harder to know who's who.",
```

Comment:
```suggestion
		desc    = "Anonymize player names and colors, making it harder to know who's who.",
```

Reply:
We are not constrained by description sizes, I would personally err on the side of being more precise rather than less 😄

Reply:
In UX terms, the more that is written, the less likely it is to be read. Brevity is best.

Reply:
For things that are displayed at all times I agree, but this is a hover description that's shown only on demand, I still think being precise is more important in this case 😅